Makefile

```diff
@@ -32,6 +32,10 @@ tosca-cpp:
 		-DTOSCA_ASAN="$(TOSCA_CPP_ASAN)"; \
 	cmake --build build --parallel
 
+test-cpp-coverage: TOSCA_CPP_BUILD = Debug
```

Comment:
should this not also be `tosca-cpp-coverage` ?? 

also, should we try to keep it consistent with the other coverage commands, like `ct-coverage-lfvm`. wdyt?

Reply:
I don't think so:

- tosca-cpp: job compiles c++ code
- test-cpp: job compiles and runs c++ unit tests
- test-cpp-coverage: same but with coverage

I will introduce ct-coverage-evmzero in the next pr.